src/client/components/HealthCareApp.jsx

```diff
@@ -158,20 +158,23 @@ class HealthCareApp extends React.Component {
     if (submissionStatus === false) {
       submitButton = (
         <ProgressButton
+            additionalClass="submit-status-not-submitted"
```

Comment:
Did you add this in order to figure out what "state" the button is in for testing? You should be able to do that with Nightwatch's `text` expectation and use that to see what the text of the button is, since that will indicate the state.